src/python/dxpy/scripts/dx.py

```diff
@@ -2514,7 +2514,13 @@ def install(args):
 def uninstall(args):
     app_desc = get_app_from_path(args.app)
     if app_desc is None:
-        parser.exit(1, 'Could not find the app\n')
+        #in case we lost permission for this app, try to remove it by Id.
+        try:
+            dxpy.api.app_uninstall(args.app)
+            print('Uninstalled application with id: ' + args.app)
```

Comment:
I think this print statement can be the similar to the original:
`print('Uninstalled the ' + args.app + ' app')`